### ISSUE TITLE
Frage bzgl. User-Scoping von /v1/resources

### ISSUE THREAD

**Issue by kremer-io** [hpi-schul-cloud/resources-api-v1#101]
Sieht die aktuelle Spezifikation vor, dass `/v1/resources` nur Ressourcen zurückgibt, die dem Nutzer aus des Auth-Header gehören?

**Comment by niccokunzmann** [comment]
Ja. /resources ist nur für die eigenen resourcen. Wir können das auch 
ändern.

Es muss irgendwie klar sien, welche Resourcen von einem selbst sind.
